<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreeSliceWidget</class>
 <widget class="QWidget" name="ThreeSliceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10,10,1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QmitkRenderWindow" name="widget_panorama" native="true"/>
     </item>
     <item>
      <widget class="QmitkRenderWindow" name="widget_3d" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QmitkRenderWindow" name="widget_sagittal" native="true"/>
     </item>
     <item>
      <widget class="QmitkRenderWindow" name="widget_coronal" native="true"/>
     </item>
     <item>
      <widget class="QmitkRenderWindow" name="widget_axial" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkRenderWindow</class>
   <extends>QWidget</extends>
   <header>QmitkRenderWindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
